core/src/scene/sceneLoader.cpp

```diff
@@ -42,16 +42,56 @@ const std::string DELIMITER = ":";
 // TODO: make this configurable: 16MB default in-memory DataSource cache:
 constexpr size_t CACHE_SIZE = 16 * (1024 * 1024);
 
-bool SceneLoader::loadScene(const std::string& _sceneString, Scene& _scene) {
+bool SceneLoader::loadScene(const std::string& _sceneString, Scene& _scene, Node& root) {
```

Comment:
I'm unclear why this method now takes a `Node&` argument, there doesn't seem to be anything functionally different going on here.